src/robusta/runner/config_loader.py

```
@@ -84,32 +84,31 @@ def __reload_scheduler(self, playbooks_registry: PlaybooksRegistry):
-            self.__create_receiver()
-
-    def __create_receiver(self):
-        robusta_sinks = self.registry.get_sinks().get_robusta_sinks()
-        if not robusta_sinks:
-            logging.info("No robusta sinks found, skipping receiver creation")
-            return
-
-        robusta_sink = robusta_sinks[0]
+            self.__create_receiver(robusta_sink)
 
+    def __create_receiver(self, robusta_sink: "RobustaSink"):
```

Ruff (0.11.9)

Undefined name `RobustaSink`

(F821)